### PR TITLE
Xabibullo best student | car shop project

### PR DESCRIPTION
**Pull request by Xabibullo01**

### CODE REVIEW

File: Xabibullo_the_best_student/main.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+def return_price(color_index, type_index, car_types, car_colors, color_prices):
+    base_price = 20000
+    type_price_modifier = 1000 * (type_index + 1)
+    color_price_modifier = color_prices[color_index]  
+    total_price = base_price + type_price_modifier + color_price_modifier
+    return f"{car_colors[color_index]} {car_types[type_index]}", total_price
+
+car_description, price = return_price(selected_color, selected_type, car_types, car_colors, color_prices)
+print(f"Tanlangan avtomobil: {car_description}\nNarxi: ${price}")
```
Comment (Shaxzodbek16) [inline Xabibullo_the_best_student/main.py:48]: you have no any class, add some classes to complete this task

File: Xabibullo_the_best_student/main.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+car_types = ["Sedan", "Hatchback", "Coupe", "Minivan", "SUV"]
+print("""
+1. Sedan
+2. Hatchback
+3. Coupe 
+4. Minivan
+5. SUV
+""")
+
+while True:
```
Comment (IslomboevBotir) [inline Xabibullo_the_best_student/main.py:10]: This task isn`t completed. Retry write logic again

File: Xabibullo_the_best_student/main2.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+selected_car = car_type_selector.choose_car_type()
+
+car_model_selector = CarModel(selected_car)
+selected_model = car_model_selector.choose_model()
+
+car_color_selector = CarColor()
+selected_color = car_color_selector.choose_color()
+
+final_price = PriceCalculator.calculate_price(selected_model, selected_color)
+Summary.display(selected_car, selected_model, selected_color, final_price)
```
Comment (Shaxzodbek16) [inline Xabibullo_the_best_student/main2.py:137]: write main func and __name__ __main__ check.

File: Xabibullo_the_best_student/oop11.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+class Women:
```
Comment (Shaxzodbek16) [inline Xabibullo_the_best_student/oop11.py:1]: Why you write Women class?

**Comment by Shaxzodbek16** [comment]
The best student

**Pull request merged**